staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -29,11 +29,19 @@ const (
 
 	// owner: @tallclair
 	// alpha: v1.5
+	// beta: v1.6
 	//
 	// StreamingProxyRedirects controls whether the apiserver should intercept (and follow)
 	// redirects from the backend (Kubelet) for streaming requests (exec/attach/port-forward).
 	StreamingProxyRedirects utilfeature.Feature = "StreamingProxyRedirects"
 
+	// owner: @tallclair
+	// alpha: v1.9
```

Comment:
BTW - I put 1.9 here because I think we should cherrypick this back to 1.9

Reply:
Still? :)